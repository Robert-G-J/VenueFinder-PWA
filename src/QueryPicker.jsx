import React from "react";

const QueryPicker = ({ venues, id }) => (
  <div className="picker">
    <ul className="picker__list">
      {venues.map((venue, index) => (
        <li key={`item-${index}`} className="picker__item">
          <button>{venue.name}</button>
        </li>
      ))}
    </ul>
  </div>
);

export default QueryPicker;
