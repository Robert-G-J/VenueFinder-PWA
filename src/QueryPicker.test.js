import React from "react";
import QueryPicker from "./QueryPicker";
import renderer from "react-test-renderer";

describe("The Query Picker component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<QueryPicker venues={[]} id={1} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
