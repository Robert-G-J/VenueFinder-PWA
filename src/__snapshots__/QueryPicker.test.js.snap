// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Query Picker component renders correctly 1`] = `
<div
  className="picker"
>
  <ul
    className="picker__list"
  />
</div>
`;
